assets/css/login.css

```diff
@@ -0,0 +1,9 @@
+/**
+ * Login page
+ */
+.page--login {}
+
+.page--login .cd-grid > * {
+  display: block;
+  background: transparent;
+}
```

Comment:
When I created the grid, I noticed that the inner contents of my two grid columns took on a flexbox layout which was quite disruptive until I unset it. I did review the [CD Grid demo](https://feature.commondesign-unocha-org.ahconu.org/demo#cd-grid) and the styles are the same there as this branch, with the grid items using a flex layout with `row nowrap` and having a gray background.

Is my workaround the simplest fix at the moment?

Reply:
Ya, if it works, stick with it. The cd-grid is used pretty sparingly in other implementations so this is a good opportunity to tweak it without too much upset, but I'm not sure when I'll be able to get to it.